tailormap-components/projects/core/src/lib/user-interface/attributelist/attributelist-tab-toolbar/attributelist-tab-toolbar.component.ts

```diff
@@ -41,8 +41,7 @@ export class AttributelistTabToolbarComponent implements OnInit {
     this.exportParams.type = format;
     this.exportService.exportFeatures(this.exportParams).subscribe((response => {
       window.location.href = response.url;
-    }), (error) => console.log('Error downloading the export:\n  ' + error.message),
-      () => console.info('File exported successfully'));
+    }), (error) => window.alert('Error downloading the export:\n  ' + error.message));
```

Comment:
Hier snackbar gebruiken

Reply:
Zal ik naar kijken. Vanwege het "verbod" op console.log had ik er eerst snel even een alert van gemaakt.

Reply:
Is aangepast